Products/Zuul/catalog/events.py

```diff
@@ -64,7 +64,8 @@ def onIndexingEvent(ob, event):
 
     if idxs:
         s_idxs = set(idxs)
-        obj_indexes = set(model_catalog.get_indexes(object_to_index))
+        obj_indexes = reduce(lambda indexed, not_indexed: indexed.union(not_indexed),
```

Comment:
This is not very readable to me. No need to create a lambda and make an additional call to reduce.